### ISSUE TITLE
Warning from the Material for MkDocs team

### ISSUE THREAD

**Issue by SamuelLarkin** [EveryVoiceTTS/EveryVoice#811]
### Bug description

This is a warning for an upcoming change that seems to have no compatibility

### How to reproduce the bug

```sh
mkdocs build
```

### Error messages and logs

```
 │  ⚠  Warning from the Material for MkDocs team
 │
 │  MkDocs 2.0, the underlying framework of Material for MkDocs,
 │  will introduce backward-incompatible changes, including:
 │
 │  × All plugins will stop working – the plugin system has been removed
 │  × All theme overrides will break – the theming system has been rewritten
 │  × No migration path exists – existing projects cannot be upgraded
 │  × Closed contribution model – community members can't report bugs
 │  × Currently unlicensed – unsuitable for production use
 │
 │  Our full analysis:
 │
 │  https://squidfunk.github.io/mkdocs-material/blog/2026/02/18/mkdocs-2.0/
```

### Environment

<details>
<summary>Current environment</summary>
<pre>
EveryVoice Diagnostic information
EveryVoice version: 0.5.0
Python version: 3.12.7 (main, Oct 16 2024, 04:37:19) [Clang 18.1.8 ]
System: Linux 5.14.0-570.44.1.el9_6.x86_64 #1 SMP PREEMPT_DYNAMIC Tue Sep 9 05:17:30 EDT 2025 x86_64

*torch* modules installed using pip:
pytorch-lightning==2.6.1
torch==2.7.1
torchaudio==2.7.1
torchinfo==1.8.0
torchmetrics==1.9.0

Other modules installed using pip:
absl-py==2.4.0
aiohappyeyeballs==2.6.1
aiohttp==3.13.5
aiosignal==1.4.0
annotated-doc==0.0.4
annotated-types==0.7.0
anyio==4.13.0
anytree==2.13.0
attrs==26.1.0
audioread==3.1.0
babel==2.18.0
backrefs==7.0
brotli==1.2.0
certifi==2026.4.22
cffi==2.0.0
charset-normalizer==3.4.7
chevron==0.14.0
click==8.1.8
clipdetect==0.1.4
colorama==0.4.6
coloredlogs==15.0.1
contourpy==1.3.3
cycler==0.12.1
decorator==5.2.1
deepdiff==9.0.0
dnspython==2.8.0
einops==0.5.0
einops-exts==0.0.4
email-validator==2.3.0
et-xmlfile==2.0.0
-e file:///gpfs/home/larkins/git/EveryVoice
fastapi==0.135.1
filelock==3.29.0
fonttools==4.62.1
frozenlist==1.8.0
fsspec==2026.3.0
g2p==2.3.1
ghp-import==2.1.0
gradio==6.13.0
gradio-client==2.5.0
grapheme==0.6.0
griffelib==2.0.2
groovy==0.1.2
grpcio==1.80.0
h11==0.16.0
hf-gradio==0.4.1
hf-xet==1.4.3
hjson==3.1.0
httpcore==1.0.9
httpx==0.28.1
huggingface-hub==1.12.0
humanfriendly==10.0
idna==3.13
ilt-editdistance==0.8.2
ilt-panphon==0.20.3
iniconfig==2.3.0
ipatok==0.4.2
jinja2==3.1.6
joblib==1.5.3
kiwisolver==1.5.0
lazy-loader==0.5
librosa==0.11.0
lightning==2.6.1
lightning-utilities==0.15.3
llvmlite==0.47.0
loguru==0.6.0
lxml==6.1.0
markdown==3.10.2
markdown-it-py==4.0.0
markupsafe==3.0.3
matplotlib==3.10.9
mdurl==0.1.2
merge-args==0.1.5
mergedeep==1.3.4
mike==2.2.0
mkdocs==1.6.1
mkdocs-autorefs==1.4.4
mkdocs-click==0.9.0
mkdocs-get-deps==0.2.2
mkdocs-macros-plugin==1.5.0
mkdocs-material==9.7.6
mkdocs-material-extensions==1.3.1
mkdocs-typer==0.0.3
mkdocstrings==1.0.4
mkdocstrings-python==2.0.3
monotonic-align @ git+https://github.com/resemble-ai/monotonic_align.git@c6e5e6cb19882164027eb6e35118e841eed9298e
mpmath==1.3.0
msgpack==1.1.2
multidict==6.7.1
munch==4.0.0
munkres==1.1.4
networkx==3.6.1
nltk==3.9.3
numba==0.65.1
numpy==2.4.4
nvidia-cublas-cu12==12.6.4.1
nvidia-cuda-cupti-cu12==12.6.80
nvidia-cuda-nvrtc-cu12==12.6.77
nvidia-cuda-runtime-cu12==12.6.77
nvidia-cudnn-cu12==9.5.1.17
nvidia-cufft-cu12==11.3.0.4
nvidia-cufile-cu12==1.11.1.6
nvidia-curand-cu12==10.3.7.77
nvidia-cusolver-cu12==11.7.1.2
nvidia-cusparse-cu12==12.5.4.2
nvidia-cusparselt-cu12==0.6.3
nvidia-nccl-cu12==2.26.2
nvidia-nvjitlink-cu12==12.6.85
nvidia-nvtx-cu12==12.6.77
openpyxl==3.1.5
orderly-set==5.5.0
orjson==3.11.8
packaging==26.2
paginate==0.5.7
pandas==2.3.3
pathspec==1.1.1
pillow==12.2.0
platformdirs==4.9.6
pluggy==1.6.0
pooch==1.9.0
prompt-toolkit==3.0.52
propcache==0.4.1
protobuf==7.34.1
pycparser==3.0
pydantic==2.7.4
pydantic-core==2.18.4
pydub==0.25.1
pydub-ng==0.2.0
pygments==2.20.0
pymdown-extensions==10.21.3
pympi-ling==1.71
pyparsing==3.3.2
pytest==7.4.4
python-dateutil==2.9.0.post0
python-multipart==0.0.26
python-slugify==5.0.0
pytz==2026.1.post1
pyworld-prebuilt==0.3.4.4
pyyaml==6.0.3
pyyaml-env-tag==1.1
questionary==2.1.1
readalongs==1.2.2
regex==2026.4.4
requests==2.33.1
rich==15.0.0
safehttpx==0.1.7
safetensors==0.7.0
scikit-learn==1.8.0
scipy==1.17.1
semantic-version==2.10.0
setuptools==79.0.1
shellingham==1.5.4
simple-term-menu==1.5.2
six==1.17.0
soundfile==0.13.1
soundswallower==0.6.7
soxr==1.0.0
starlette==1.0.0
super-collections==0.6.2
sympy==1.14.0
tabulate==0.9.0
tensorboard==2.20.0
tensorboard-data-server==0.7.2
termcolor==3.3.0
text-unidecode==1.3
threadpoolctl==3.6.0
tokenizers==0.22.2
tomlkit==0.14.0
tqdm==4.67.3
transformers==5.7.0
triton==3.3.1
typer==0.23.1
typing-extensions==4.15.0
typing-inspection==0.4.2
tzdata==2026.2
urllib3==2.6.3
uvicorn==0.46.0
verspec==0.1.0
watchdog==6.0.0
wcwidth==0.6.0
webvtt-py==0.5.1
werkzeug==3.1.8
yarl==1.23.0
yaspin==3.4.0
</pre>
</details>

**Comment by SamuelLarkin** [comment]
FYI I discovered this warning in a different project.  I'll have to fix it it that other project so might as well fix it for EveryVoice once I have the solution.

**Comment by joanise** [comment]
Short term mitigation: we can lock `mkdocs<2` until we have a better fix.
